src/widgets/HomePageWidgets/Hero/components/HeroSlider.tsx

```diff
@@ -0,0 +1,26 @@
+import { IHeroData } from '../model/types';
+
+export const HeroSlider = ({ hero }: IHeroData) => {
+  console.log('hero from HeroSlider', hero);
```

Comment:
remove all console logs

@BohdanTaran please add the rule to eslint that restricts all console log
https://eslint.org/docs/latest/rules/no-console